provider/provider.go

```diff
@@ -217,7 +217,7 @@ func DatabricksProvider() terraform.ResourceProvider {
 			"azure_use_pat_for_cli": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     true,
+				Default:     false,
```

Comment:
quick question does this mean we do not make pat token by default and the user has to set this to true?

Reply:
only for Azure CLI. SP behavior is not changed.

Overall customers create one-hour PAT tokens very often, even though Azure requires them only for create secret scopes method.

Reply:
ahh okay thx just wanted some clarification LGTM. Unfortunately the api error message here is not so great it will say api is unavailable.